src/client/datascience/jupyter/kernels/kernel.ts

```diff
@@ -329,9 +351,17 @@ export class Kernel implements IKernel {
                             this.notebookDocument,
                             placeholderCellPromise
                         );
+                        // if (!process.env.XYZ1234) {
```

Comment:
?

Reply:
I've left that for testing, will remove.
I thought my comments were on github, hmm

Reply:
Fixed